pkgs/dart_mcp/example/workflow_client.dart

```diff
@@ -131,6 +148,36 @@ final class WorkflowClient extends MCPClient with RootsSupport {
   final gemini.GenerativeModel model;
   final bool verbose;
 
+  Sink<String>? _createLogSink(File? logFile) {
+    if (logFile == null) {
+      return null;
+    }
+    Sink<String>? logSink;
+    logFile.createSync(recursive: true);
+    final fileByteSink = logFile.openWrite(
```

Comment:
Just to confirm, this will actually write the bytes to disk relatively quickly when we write to the sink? Do we need to call `flush` or anything?

Reply:
When I did this previously I just did regular writes in append mode for each line that came in, instead of keeping it open like this. In theory this is better though, but we do want to actually see the logs.

Reply:
Flushing should do that, and this is only in the example.

I did look at doing the `flush` where we do the adding to the sink in the main package, but then we'd have to change the `Sink<String>` to an `IOSink`, since the base `Sink` doesn't have `flush()`, which would be a breaking change.